I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/fixBug.rpy", line 27, in script
    return()
Exception: Could not find return label ('game/script.rpy', 1747798971, 11).

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/fixBug.rpy", line 27, in script
    return()
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\ast.py", line 1506, in execute
    next_node(renpy.game.context().lookup_return(pop=True))
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\execution.py", line 744, in lookup_return
    raise Exception("Could not find return label {!r}.".format(self.return_stack[-1]))
Exception: Could not find return label ('game/script.rpy', 1747798971, 11).

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
datingsim 1.0
Tue May 20 22:26:57 2025
